src/main/resources/validationrules/iac/iac-judicial-org-role-mapping.drl

```diff
@@ -258,3 +259,37 @@ then
       .build());
       logMsg("Rule : iac_fee_paid_judge_org_role");
 end;
+
+/*
+ * IAC "judge" org role can be created by any existing judicial office holder having
+ * "IAC Tribunal Judge (Fee-Paid)" business role.
+ */
+rule "iac_judge_org_role_for_fee_paid_judge"
+when
+  $f:  FeatureFlag(status && flagName == FeatureFlagEnum.IAC_JRD_1_0.getValue())
```

Comment:
Do we need to enable this rule with JRD or should have a separate flag for judicial booking service as this change will not part of JRD 2.0?

Reply:
This rule won't be invoked until there is a booking object found in drool memory belonging to the same userId, we it will work only post JBS integration.